src/main/java/Screens/AnalysisController.java

```diff
@@ -0,0 +1,52 @@
+package Screens;
+
+import analysis_use_case.Analysis;
+
+public class AnalysisController {
+    private final Analysis analysis;
+
+    /**
+     * AnalysisController Constructor
+     * @param analysis the Analysis object I will perform the required analysis on
```

Comment:
Try to be more specific with parameter descriptions (e.g. Analysis is a use case interactor)